pkgs/applications/audio/oxefmsynth/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, unzip, requireFile, libX11}:
+
+stdenv.mkDerivation rec {
+  pname = "oxefmsynth";
+  version = "v1.3.5";
+
+  src = fetchFromGitHub {
+    owner = "oxesoft";
+    repo = pname;
+    rev = version;
+    sha256 = "1rk71ls33a38wx8i22plsi7d89cqqxrfxknq5i4f9igsw1ipm4gn";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
+    name = "vstsdk368_08_11_2017_build_121";
+    src = requireFile {
+      name = "${name}.zip";
+      url = "http://www.steinberg.net/en/company/developers.html";
```

Comment:
That is a html url not a zip. Also please move this into a let in block.

Reply:
Yes, I took it from airwave package. Steinberg is not hosting the zip anymore. I updated the url to be a copy of it that I found in the Internet Archive.